### ISSUE TITLE
[GraphQL] Multiple aliases for single matrix field

### ISSUE THREAD

**Issue by gregorterrill** [craftcms/cms#5008]
### Description
It doesn't seem like its currently possible to query the same matrix field multiple times (with different aliases) for different block types. I want to be able to do something like this in order to group blocks by their type:

```
alias1: matrixFieldHandle(type:"blockType1Handle") {
  ...on matrixFieldHandle_blockType1Handle_BlockType {
    fieldA
    fieldB
    fieldC
  }
}

alias2: matrixFieldHandle(type:"blockType2Handle") {
  ...on matrixFieldHandle_blockType2Handle_BlockType {
    fieldD
    fieldE
    fieldF
  }
}
```

When I try this, only the last alias (`alias2` here) listed gets populated with data and any preceding aliases (just `alias1` here) are an array containing a single empty object. If I switch the order and put `alias2` first, then it's the array/empty object and `alias1` gets populated properly with data. I'm not sure if this is a bug?

If I remove the type parameters, each alias gets all the matrix blocks from the field, but since some don't match the fragment, I get an array of mostly empty objects. The ones that do match have all the correct data though.

### Additional info

- Craft version: 3.3.5
- PHP version: 7.1.18
- Database driver & version: MySQL 5.7.21
- Plugins & versions: N/A

**Comment by andris-sevcenko** [comment]
Thanks! This is fixed and will be resolved with the next 3.4 release!

**Comment by andris-sevcenko** [comment]
This was included in the Craft 3.4-beta.4 release!
